data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,62 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition: Alpha Encounter"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		has "remembrance day: offered"
+		random = 1
```

Comment:
1) If you want to check equivalence, use `==`, not `=`, which is assignment.
2) You can still just use `random < 1` for a 1% chance.
```suggestion
		random < 1
```